app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for new_person in people:
+        new_person = Person(new_person["name"], new_person["age"])
```

Comment:
u don't need to create a local variable here
```suggestion
    for person in people:
        Person(person["name"], person["age"])
```